### DISCUSSION TITLE
Book mark

### DISCUSSION THREAD

**Discussion by jagadeeshgade008** [dailydotdev/daily#630] [Feature Requests]
![image](https://user-images.githubusercontent.com/79077571/179716213-da3a5a8d-bef0-4430-a624-63993cfbdba9.png)

instead of directly adding it to the bookmark. It would be helpful if we save based on tags.

**Comment by Abdulali97** [comment]
yep, tags, category, label  - full bookmark management 
whatever...   it's part of us now  everything in one place will be great

**Comment by idoshamun** [comment]
Hi! Let's keep the discussion here #520 :)
